pkg/kubelet/kubelet.go

```diff
@@ -461,9 +458,9 @@ func NewMainKubelet(kubeCfg *kubeletconfiginternal.KubeletConfiguration,
 	}
 
 	containerGCPolicy := kubecontainer.GCPolicy{
-		MinAge:             minimumGCAge.Duration,
-		MaxPerPodContainer: int(maxPerPodContainerCount),
-		MaxContainers:      int(maxContainerCount),
+		MinAge:             time.Duration(0),
+		MaxPerPodContainer: 1,
+		MaxContainers:      -1,
```

Comment:
> Is this policy used somewhere outside of GC? If not, should we remove it?

The policy is not used outside of GC, but removing it would affect the current behavior of GC.

Reply:
Initializing directly within `NewContainerGC` might be more reasonable.

Reply:
As all fields are set to static values, we may want to refactor GC code or even get rid of (some of) it if possible. For example MinAge 0 is asking for refactoring of `func (cgc *containerGC) evictableContainers`, with MaxContainer -1 `(cgc *containerGC) evictContainers` could be simplified, etc .